src/vivarium_census_prl_synth_pop/tools/jobmon.py

```diff
@@ -0,0 +1,83 @@
+import shutil
+import uuid
+from pathlib import Path
+from typing import Union
+
+from jobmon.client.tool import Tool
+from loguru import logger
+
+
+def run_make_results_workflow(
+    raw_output_dir: Path,
+    final_output_dir: Path,
+    mark_best: bool,
+    test_run: bool,
+    artifact_path: Union[str, Path],
+    queue: str,
+    peak_memory: int,
+    max_runtime: str,
+    project: str,
+) -> None:
+    """Creates and runs a jobmon workflow to build results datasets
+    from the raw data output by observers
+    """
+    logger.info(f"Starting make_results workflow {final_output_dir}")
+    wf_uuid = uuid.uuid4()
+
+    # Deal with boolean args - click either the flag or nothing, not True/False
+    # TODO: there's got to be a better way to do this?
+    mark_best_arg = "--mark-best" if mark_best else ""
+    test_run_arg = "--test-run" if test_run else ""
+
+    # Create tool
+    tool = Tool(name="vivarium_census_prl_synth_pop.make_results")
+
+    # Create a workflow
+    workflow = tool.create_workflow(name=f"make_results_workflow_{wf_uuid}")
+
+    # Create task templates
+    template_make_results = tool.get_task_template(
+        template_name="make_results_template",
+        default_compute_resources={
+            "queue": queue,
+            "cores": 1,
+            "memory": peak_memory,
+            "runtime": max_runtime,
+            "project": project,
+            "stdout": str(final_output_dir),
+            "stderr": str(final_output_dir),
```

Comment:
Are these just sending the output to that directory or does a log file get created?

Reply:
Well, this is the final output directory for the real datasets (same as always). This is just telling jobmon to send the output logs there as well.